samples/wifi/shell/overlay-zperf.conf

```diff
@@ -1,4 +1,5 @@
 CONFIG_NET_ZPERF=y
+CONFIG_NET_ZPERF_SERVER=y
```

Comment:
we now have a Zperf snippet https://github.com/nrfconnect/sdk-nrf/tree/main/snippets/zperf should we remove these overlays and switch to the snippet? We can add board specific stuff to the boards folder?